src/components/CarouselSlider/CarouselSlider.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { createBrowserHistory } from 'history';
+import PropTypes from 'prop-types';
+import {
+  Row, Col, Carousel
+} from 'react-materialize';
+import './CarouselSlider.scss';
+
+let articlePic = '';
+const history = createBrowserHistory({ forceRefresh: true });
+const linkTo = (path) => {
+  history.push(path);
+};
+
+export const CarouselSlider = props => (
+  <div>
+    <Carousel options={{ fullWidth: true, indicators: true }}>
+      { props.articles.map((article, index) => {// eslint-disable-line
+        if (!article.imgUrl) {
+          articlePic = 'https://via.placeholder.com/300?text=AuthorsHaven';
+        } else {
+          articlePic = article.imgUrl;
+        }
```

Comment:
this sets a default placeholder image if there is no featured image for an article